#ifndef PROTOINC_H_
#define PROTOINC_H_

#include "Common.pb.h"
#include "Admin.pb.h"
#include "User.pb.h"
#include "ErrorRet.pb.h"
#include "ProtoBuilding.pb.h"
#include "ProtoBattleField.pb.h"
#include "EquipShopLogic.pb.h"
#include "BagLogic.pb.h"
#include "ProtoHero.pb.h"
#include "ProtoPass.pb.h"
#include "ProtoResource.pb.h"
#include "ProtoEquip.pb.h"
#include "ProtoMission.pb.h"
#include "ProtoScience.pb.h"
#include "ProtoPush.pb.h"
#include "ProtoYellowVip.pb.h"
#include "ProtoWeapon.pb.h"
#include "ProtoMarket.pb.h"
#include "ProtoReward.pb.h"
#include "ProtoCumulation.pb.h"
#include "ProtoGovernAffair.pb.h"
#include "ProtoPkField.pb.h"
#include "ProtoNPCActivity.pb.h"
#include "ProtoActivity.pb.h"
#include "ProtoOfficial.pb.h"
#include "ProtoFogGate.pb.h"
#include "ProtoMine.pb.h"
#include "Bot.pb.h"
#include "ProtoDailyMission.pb.h"
#include "ProtoComponent.pb.h"
#include "ProtoAchievement.pb.h"
#include "ProtoFriend.pb.h"
#include "ProtoShakeDice.pb.h"
#include "ProtoNotify.pb.h"
#include "ProtoResourceCopy.pb.h"
#include "ProtoOperaBattle.pb.h"
#include "ProtoModelTrial.pb.h"
#include "ProtoCrossBusiness.pb.h"
#include "ProtoAttackHarbor.pb.h"

#endif /* PROTOINC_H_ */
